app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -33,7 +39,13 @@ class SyncIndicatorViewController @Inject constructor(
     private val lastSyncState: Preference<LastSyncedState>,
```

Comment:
`handle sync pending` sounds weird. Sounds like it's missing a word at the end. Do you want to rename `Update SyncIndicatorViewController to handle sync pending` -> `Handle pending sync status in SyncIndicatorViewController?`